pkg/client/client.go

```diff
@@ -43,7 +41,7 @@ func Client(cred nutanixClient.Credentials, options ClientOptions) (*nutanixClie
 	if cred.URL == "" {
 		cred.URL = fmt.Sprintf("%s:%s", cred.Endpoint, cred.Port)
 	}
-	cli, err := nutanixClientV3.NewV3Client(cred, debugMode(options))
```

Comment:
how does one enable debugMode with this new library?

Reply:
The debug param is not part of the prism-go-client, hence this functionality is removed for now.